zipkin-server/src/main/java/zipkin2/server/internal/throttle/LimiterMetrics.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2015-2019 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin2.server.internal.throttle;
+
+import com.netflix.concurrency.limits.Limiter.Listener;
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import zipkin2.collector.CollectorMetrics;
+
+/** Follows the same naming convention as {@link CollectorMetrics} */
+final class LimiterMetrics {
```

Comment:
I don't know if this would improve anything, but just as a hint if this implemented `MeterBinder` and was a bean, spring would automatically register it.

Reply:
ic. thx